### PR TITLE
add squeak 5.0

### PR DESCRIPTION
**Pull request by pdebruic**

### CODE REVIEW

File: build_client_image.sh [hunk 1]
```diff
@@ -152,6 +152,20 @@ case "$ST" in
     IMAGE_BASE_NAME=Squeak4.6-15102
     mv *.sources $SOURCES_PATH
     ;;
+  # Squeak-5.0
+  Squeak-5.0)
+    cd $IMAGES_PATH
+    # 4.3 stores things in a Squeak4.3 directory. 5.0 doesn't.
+    # So we mimic the behaviour of 4.3.
+    mkdir -p Squeak4.0
```
Comment (krono) [inline build_client_image.sh]: Typo?

Comment (pdebruic) [inline build_client_image.sh]: Thanks.  Fixed ;)

**Comment by fniephaus** [comment]
>This interpreter (vers. 6505) cannot read image file (vers. 6521).

Unfortunately, Squeak 5.0 requires a Spur VM. So it needs a little more than this to work :cry:

**Comment by pdebruic** [comment]
Of course you're right.  I'll add it now.

**Comment by fniephaus** [comment]
Are you sure you used the latest SqueakSSL plugin?

**Comment by pdebruic** [comment]
@fniephaus turns out I just needed the statically linked SqueakSSL plugin from the oneclick directory rather than the one Eliot Miranda was providing with the Spur VM.  Who knew?

**Comment by krono** [comment]
I knew ;) Yes, this part has to get better.

**Comment by pdebruic** [comment]
True dat.  You did know. :P

**Comment by pdebruic** [comment]
Speaking of getting better - I started on a adaptation of the build_client_image.sh that downloads the proper cog or spur VM from Eliots site rather than using the oneclick directories.  

I'm going to hook up my fork to travis so I can get it running without pestering Dale.

**Pull request merged**